samples/createInstanceFromTemplateWithOverrides.js

```diff
@@ -13,23 +13,23 @@
 // limitations under the License.
 
 /**
- * Creates GCP instance from an instance template with a new disk and machine type.
+ * Creates a Compute Engine VM instance from an instance template, but overrides the disk and machine type options in the template.
  *
  * @param {string} projectId - ID or number of the project you want to use.
  * @param {string} zone - Name of the zone you want to check, for example: us-west3-b
  * @param {string} instanceName - Name of the new instance.
- * @param {string} instanceTemplateName - Name of the instance template using for creating the new instance.
+ * @param {string} instanceTemplateName - Name of the instance template to use when creating the new instance.
  * @param {string} machineType - Machine type you want to set in following format:
  *    "zones/{zone}/machineTypes/{type_name}". For example:
  *    "zones/europe-west3-c/machineTypes/f1-micro"
  *    You can find the list of available machine types using:
  *    https://cloud.google.com/sdk/gcloud/reference/compute/machine-types/list
  * @param {string} newDiskSourceImage - Path the the disk image you want to use for your new
  *    disk. This can be one of the public images
- *    (e.g. "projects/debian-cloud/global/images/family/debian-10")
+ *    (like "projects/debian-cloud/global/images/family/debian-10")
  *    or a private image you have access to.
- *    You can check the list of available public images using:
- *    $ gcloud compute images list
+ *    You can check the list of available public images using the doc:
```

Comment:
DevSite style avoids using the shortened version "doc". Recommend something closer to the following:
"For a list of available public images, see the documentation: http://cloud.google.com/compute/docs/images"